stix2elevator/missing_policy.py

```diff
@@ -204,13 +232,29 @@ def statement_type_as_extension_properties(container, statement, property_name,
         add_confidence_property_as_extension_property(map, statement.confidence, property_name, id)
     if map:
         if statement.value:
-            map["value"] = str(statement.value)
+            value_as_string = str(statement.value)
+            if is_literal:
+                if value_as_string in mapping:
+                    map["value"] = mapping[value_as_string]
+                else:
+                    map["value"] = convert_to_stix_literal(value_as_string)
+            else:
+                map["value"] = value_as_string
         container[property_name] = [map] if is_list else map
     else:
-        container[property_name] = [str(statement.value)] if is_list else str(statement.value)
+        if statement.value:
+            value_as_string = str(statement.value)
+            if is_literal:
+                if value_as_string in mapping:
+                    converted_value = mapping[value_as_string]
+                else:
+                    converted_value = convert_to_stix_literal(value_as_string)
+                container[property_name] = [converted_value] if is_list else converted_value
+            else:
+                container[property_name] = [value_as_string] if is_list else value_as_string
```

Comment:
Could this whole `if map... else` structure be simplified? 
```suggestion
    if statement.value:
        value_as_string = str(statement.value)
        if is_literal:
            if value_as_string in mapping:
                converted_value = mapping[value_as_string]
            else:
                converted_value = convert_to_stix_literal(value_as_string)
        else:
            converted_value = value_as_string
            
        if map:
            map["value"] = converted_value
            container[property_name] = [map] if is_list else map
        else:
            container[property_name] = [converted_value] if is_list else converted_value
```

Reply:
testing